### PR TITLE
[MP v2] Make test suite green

### PR DESCRIPTION
**Pull request by alloy**
💚

### CODE REVIEW

File: __mocks__/react-relay.js [hunk 1]
```diff
@@ -2,13 +2,20 @@
 // Should be used in tests as a replacement for `react-storybooks-relay-container`
 //
 var React = require.requireActual("react")
-var Relay = require.requireActual("react-relay")
+
+const graphql = (strings, ...keys) => {
```
Comment (damassi) [inline __mocks__/react-relay.js]: Mind adding a small comment about what this does?

Comment (alloy) [reply __mocks__/react-relay.js]: Done :+1:  https://github.com/artsy/reaction/pull/3002/commits/6e0b7f2873fb8ede6bfd8dbf29b30200330b2662

**Comment by alloy** [comment]
The [previous build](https://circleci.com/workflow-run/2689453a-cecb-4339-a971-7c0373d1c2a2) was all green, but I then added a comment with the `[skip ci]` label and I guess that means it goes away entirely here 🤷‍♂

**Comment by artsyit** [comment]
:rocket: PR was released in v22.0.0 :rocket:

**Pull request merged**